src/style/calc_dimension.rs

```diff
@@ -0,0 +1,54 @@
+//! Dimensions that are have to be calculated while resolving.
+
+use crate::resolve::MaybeResolve;
+
+use super::Dimension;
+
+/// A [`Dimension`] calculation.
```

Comment:
This could be clearer. We're not far off restating the type name here, which is never a good sign :p

Reply:
Fair, I have reworked that :D